llvm/test/MC/AMDGPU/gfx11_asm_vop2.s

```diff
@@ -1,4 +1,4 @@
-// NOTE: Assertions have been autogenerated by utils/update_mc_test_checks.py UTC_ARGS: --version 5
+// NOTE: Assertions have been autogenerated by utils/update_mc_test_checks.py UTC_ARGS: --unique --version 5
```

Comment:
Do we actually want to check in the test files with --unique in the RUN line, or just run the update script with the option?@jayfoad

Reply:
What I said was misleading. I mean, should we pass --unique to the update script? 
Actually, I think it is probably good as is. I can't see how it would be helpful to have duplicate tests, and having --unique will enforce that on any future change.

Reply:
Thanks! I think for now the script will just put `--unique` to the headline and it does not force people to use `--unique` for the next run. But it will show up in diff so that people will realize that they should add `--unique` which I think should be good enough